sympy/integrals/tests/test_laplace.py

```diff
@@ -142,24 +141,35 @@ def test_laplace_transform():
     assert (LT(sinh(2*sqrt(a*t)), t, s) ==
             (sqrt(pi)*sqrt(a)*exp(a/s)/s**(S(3)/2), 0, True))
     assert (LT(sqrt(t)*sinh(2*sqrt(a*t)), t, s, simplify=True) ==
-            ((-sqrt(a)*s**(S(5)/2) + sqrt(pi)*s**2*(2*a + s)*exp(a/s) *
-              erf(sqrt(a)*sqrt(1/s))/2)/s**(S(9)/2), 0, True))
+            ((sqrt(a)*sqrt(s) + sqrt(pi)*(2*a + s)*exp(a/s)*
+              erf(sqrt(a)/sqrt(s))/2)/s**(S(5)/2), 0, True))
     assert (LT(sinh(2*sqrt(a*t))/sqrt(t), t, s) ==
-            (sqrt(pi)*exp(a/s)*erf(sqrt(a)*sqrt(1/s))/sqrt(s), 0, True))
+            (sqrt(pi)*exp(a/s)*erf(sqrt(a)/sqrt(s))/sqrt(s), 0, True))
     assert (LT(sinh(sqrt(a*t))**2/sqrt(t), t, s) ==
-            (sqrt(pi)*(exp(a/s) - 1)/(2*sqrt(s)), 0, True))
+            (sqrt(pi)*(2*exp(a/s) - 2)/(4*sqrt(s)), 0, True))
```

Comment:
Please don't change the test cases like this. This should be, for example,

```
assert (LT(sinh(sqrt(ar*t))**2/sqrt(t), t, s) ==
    (sqrt(pi)*(exp(a/s) - 1)/(2*sqrt(s)), Max(0, -a), True))
```

or similar.